bin/add-toc.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# ==============================================================================
+#/ This script recursively scans all markdown files in a given directory and
+#/ replaces all occurrences of "<!-- toc -->" with an actual table of content.
+#/
+#/ Requires the markdown-toc node module to be available in the project.
+# ------------------------------------------------------------------------------
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+shortUsage() {
```

Comment:
We created a shell script before here with all the guidelines included. The script is here:
https://github.com/DealerDirect/php-qa-tools/blob/master/bin/install.sh

- There are no descriptions above the functions.